app/views/govuk_component/search.raw.html.erb

```diff
@@ -0,0 +1,21 @@
+<%
+  appearance_class ||= "on-light-background"
+  layout_class ||= ""
+  search_term ||= ""
```

Comment:
Should we mirror HTML where possible so this would be 'search_value' or 'value'?